### PR TITLE
SONAR-7249 Identity provider plugin must provide the user login

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

**Comment by julienlancelot** [comment]
Associated PR on Github is here : https://github.com/SonarSource/sonar-auth-github/pull/2

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protobuf' package (org.sonarsource.sonarqube:sonar-db:target/generated-sources/protobuf/org/sonar/db/protobuf) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

File: sonar-db/src/main/java/org/sonar/db/version/v54/MigrateDisabledUsersToOnlyKeepLoginAndName.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      "AND external_identity IS NOT NULL " +
+      "AND external_identity_provider IS NOT NULL " +
+      "AND crypted_password IS NOT NULL " +
+      "AND salt IS NOT NULL " +
+      "AND remember_token IS NOT NULL " +
+      "AND remember_token_expires_at IS NOT NULL "
+    ).setBoolean(1, false);
+    update.update("UPDATE users SET email=null, scm_accounts=null, " +
+      "external_identity=null, external_identity_provider=null, " +
+      "crypted_password=null, salt=null, " +
```
Comment (SonarTech) [inline sonar-db/src/main/java/org/sonar/db/version/v54/MigrateDisabledUsersToOnlyKeepLoginAndName.java]: CRITICAL Remove this hard-coded password. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2068)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protobuf' package (org.sonarsource.sonarqube:sonar-db:target/generated-sources/protobuf/org/sonar/db/protobuf) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Pull request merged**